src/Peering/Peering/ChangeLog.md

```diff
@@ -19,6 +19,12 @@
 -->
 ## Upcoming Release
 
+## Version 0.2.0
```

Comment:
@rileymckenna you can remove this version header and just put everything under `## Upcoming Release` -- we will do our own versioning before we release the module